test/TrialsTests.jl

```diff
@@ -233,4 +233,17 @@ else
 
 end
 
+trial = BenchmarkTools.Trial(BenchmarkTools.Parameters(), [1.0, 1.01], [0.0, 0.0], 0, 0)
```

Comment:
Should this be its own commit?

Reply:
I'm not sure what you mean. This test is diagnostic against the current and the previous `remtrailingzeros` bugfixes by checking that "1.01 ns" is not printed as "1.010 ns" or "11 ns". It is also diagnostic against the bugfix for compability with Julia 1.0 by calling `show` on a `Trial`. It seems better to commit them together.

Reply:
Ah right, I just thought this was a random test.